docs/output.md

```diff
@@ -7,34 +7,39 @@ This document describes the output produced by the pipeline.
 The pipeline is built using [Nextflow](https://www.nextflow.io/)
 and processes data using the following steps:
 
-* [Arriba](#arriba)
-* [EricScript](#ericscript)
-* [FastQC](#fastqc) - read quality control
-* [FusionCatcher](#fusioncatcher)
-* [FusionInspector](#fusion-inspector)
-* [fusion-report](#fusion-report)
-* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
-* [Pizzly](#pizzly)
-* [Squid](#squid)
-* [Star-Fusion](#star-fusion)
+- [nf-core/rnafusion: Output](#nf-corernafusion-output)
```

Comment:
Can you add `<!-- omit in toc -->` to the header, that way it won't be in the automatic TOC?

Reply:
done